interface/modules/custom_modules/oe-module-claimrev-connect/public/claims.php

```diff
@@ -0,0 +1,334 @@
+                    </div>                    
+                    <div class="col">
+                        
+                    </div>
+                </div>
+                <div class="row">
+                    <div class="col">
+                        <div class="form-group">
+                            <label for="patFirstName">Patient First Name</label>
+                            <input type="text" class="form-control"  id="patFirstName" name="patFirstName"  value="<?php echo isset($_POST['patFirstName']) ? $_POST['patFirstName'] : '' ?>"  placeholder="Patient First Name"/>
+                        </div>
+                    </div>
+                    <div class="col">
+                        <div class="form-group">
+                            <label for="patLastName">Patient Last Name</label>
+                            <input type="text" class="form-control"  id="patLastName" name="patLastName"  value="<?php echo isset($_POST['patLastName']) ? $_POST['patLastName'] : '' ?>" placeholder="Patient Last Name"/>
+                        </div>
+                    </div>
+                    <div class="col">
+                    
+                    </div>
+                    <div class="col">
+                    
+                    </div>
+                </div>   
+                <div class="row">
+                    <div class="col">
+                        <button type="submit" name="SubmitButton" class="btn btn-primary">Submit</button>
+                    </div>
+                    <div class="col-10">
+                    
+                    </div>
+                </div>            
+                
+            </div> 
+        </form>
+
+
+        <?php                  
+            $datas = null;
+            if(isset($_POST['SubmitButton'])) { //check if form was submitted
+
+                $datas = ClaimsPage::SearchClaims($_POST);
+                
+
+            }
+            
+            if($datas != null){ ?>
+                <table class="table">
+                <thead>
+                    <tr>
+                        <th scope="col">Status</th>
+                        <th scope="col">Payer Info</th>
+                        <th scope="col">Provider Info</th>
+                        <th scope="col">Patient Info</th>
+                        <th scope="col">Claim Info</th>                 
+                    </tr>
+                </thead>
+                <tbody>
+                    <?php
+                        foreach($datas as $data) 
+                        {
+                    ?>                            
+                        <tr>
+                            <td>
+                                <div class="row">
+                                    <div class="col">
+                                        <div class="row">
+                                            <div class="font-weight-bold col">
+                                                ClaimRev Status:
```

Comment:
xlt(), note the colon goes outside the translation constant:
```
<?php echo xlt("ClaimRev Status"); ?>:
```

Reply:
same as above for lines 147, 161, 173, 187, 195, 203, 213, 221, 231, 239, 247, 255, 265, 273, 281, 289, 297